packages/site/src/components/ComputeControls.tsx

```diff
@@ -0,0 +1,126 @@
+import { Spinner } from './Spinner';
+import {
+  PlayCircleIcon,
+  ArrowPathIcon,
+  MinusCircleIcon,
+  ArrowTopRightOnSquareIcon,
+} from '@heroicons/react/24/outline';
```

Comment:
Can you change all instances of these to default imports!

Otherwise remix bundles all of the icons. :(

```
import ArrowTopRightOnSquareIcon from '@heroicons/react/24/outline/ArrowTopRightOnSquareIcon';
```